README.md

```diff
@@ -1,16 +1,32 @@
-Welcome to FAIRChem! (Under construction - 2024-05-06~2024-05-07)
+## `fairchem` by FAIR Chemistry
+<p align="center">
+  <img width="559" height="200" src="https://github.com/FAIR-Chem/fairchem/assets/45150244/d3abd756-cb96-40eb-b3a8-976939a2f0d8"?
+</p>
 
-(2024-05-06) Repository changes
+`fairchem` is the [FAIR](https://ai.meta.com/research/) Chemistry's centralized repository of all its data, models, demos, and application efforts for materials science and quantum chemistry.
 
-To better test integration between our packages and keep them up to date we have migrated all existing repositories under Open-Catalyst-Project to a single repository (this one), FAIRChem. 
+### Documentation
+If you are looking for `Open-Catalyst-Project/ocp`, it can now be found at [`fairchem.core`](src/fairchem/core). Visit its corresponding documentation [here](https://fair-chem.github.io/).
 
-Our new structure supports the fairchem package namespace with several independently installable packages:
-* [fairchem.core](src/fairchem/core) (formerly [OpenCatalystProject - OCP](https://github.com/Open-Catalyst-Project/ocp/tree/main))
-* [fairchem.data.oc](src/fairchem/data/oc) (fomerly [Open-Catalyst-Dataset](https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset))
-* [fairchem.data.om](src/fairchem/data/om) (formerly [om-data](https://github.com/Open-Catalyst-Project/om-data))
-* [fairchem.data.odac](src/fairchem/data/odac) (formerly [odac-data](https://github.com/Open-Catalyst-Project/odac-data/tree/main))
-* [fairchem.demo.ocpapi](src/fairchem/demo/ocpapi) (formerly [ocpapi](https://github.com/Open-Catalyst-Project/ocpapi))
-* [fairchem.applications.adsorbml](src/fairchem/applications/AdsorbML) (formerly [AdsorbML](https://github.com/Open-Catalyst-Project/AdsorbML))
-* [fairchem.applications.cattsunami](src/fairchem/applications/CatTSunami)
+### Contents
+The repository is organized into several directories to help you find what you are looking for:
 
-[go to the docs!](https://open-catalyst-project.github.io/ocp/)
+- [`fairchem.core`](src/fairchem/core): State of the art machine learning models for materials science and chemistry
+- [`fairchem.data`](src/fairchem/data): Dataset downloads and input generation codes
+- [`fairchem.demo`](src/fairchem/demo): Python API for the [Open Catalyst Demo](https://open-catalyst.metademolab.com/)
+- [`fairchem.applications`](src/fairchem/applications): Follow up applications and works (AdsorbML, CatTSunami, etc.)
+  
+### Installation
+Packages can be installed in your environment by the following:
+```
+pip install -e packages/fairchem-{package-to-download}
```

Comment:
packages/fairchem-{package-to-download} 
I don't have a better idea really, {package-to-download} seems ambiguous to me, 
maybe, 
packages/{fairchem-package-name}